Python/chapter02/2.4 - Partition/miguel_2.4_soln.py

```diff
@@ -92,7 +92,7 @@ def pop_head(self) -> Optional[Node]:
 
     def append(self, ll: object):
```

Comment:
```suggestion
    def append(self, ll: LinkedList) -> None:
```

that way you don't even need the `isinstance`.

Reply:
I get a `NameError: name 'LinkedList' is not defined` exception when doing this.

Reply:
oh right, i always forget about that quirk.

```suggestion
     def append(self, ll: 'LinkedList') -> None:
```

(note the quotes).